reflex/custom_components/custom_components.py

```diff
@@ -10,15 +10,35 @@
 from contextlib import contextmanager
 from pathlib import Path
 
+import click
 import httpx
-import typer
 
 from reflex import constants
 from reflex.config import get_config
 from reflex.constants import CustomComponents
 from reflex.utils import console
 
-custom_components_cli = typer.Typer()
+
+@click.group
+def custom_components_cli():
+    """CLI for creating custom components."""
+    pass
+
+
+def _set_loglevel_from_cli(loglevel: str | None):
+    console.set_log_level(
+        constants.LogLevel.from_string(loglevel) or get_config().loglevel
+    )
+
+
+loglevel_option = click.option(
```

Comment:
we should use `callback` and `is_eager` to automatically handle setting the log level, so that the other commands don't need to do anything special